proxy_agent_extension/src/linux.rs

```diff
@@ -5,14 +5,28 @@ use std::thread;
 #[cfg(not(windows))]
 use std::time::Duration;
 
+use crate::logger;
 use crate::service_main;
+use proxy_agent_shared::{misc_helpers, telemetry};
 
 #[cfg(not(windows))]
 pub fn start_service_wait() {
     // start service
     let service_state = service_main::service_state::ServiceState::new();
     service_main::run(service_state);
-
+    let message = format!(
```

Comment:
already logged in service_main.rs and proxy_agent_extension\src\handler_main.rs file

Reply:
reverted to original since this is redundant logging